mail_attach_existing_attachment/wizard/mail_compose_message.py

```diff
@@ -44,14 +44,14 @@ def default_get(self, fields_list):
         relation='mail_compose_message_ir_attachments_object_rel',
         column1='wizard_id', column2='attachment_id', string='Attachments')
 
-    @api.model
-    def get_mail_values(self, wizard, res_ids):
-        res = super(MailComposeMessage, self).get_mail_values(wizard, res_ids)
-        if wizard.object_attachment_ids.ids and wizard.model and\
+    @api.multi
+    def get_mail_values(self, res_ids):
+        res = super(MailComposeMessage, self).get_mail_values(res_ids)
+        if self.object_attachment_ids.ids and self.model and\
```

Comment:
But when calling super class there is an ensure_one() so i think that i don't need it